### ISSUE TITLE
"TypeError: Cannot read property 'test' of undefined" after running testcafe tests using node runner

### ISSUE THREAD

**Issue by jiteshsojitra** [DevExpress/testcafe#4669]
After installing TestCafe 1.8.0, i have started getting error of "TypeError: Cannot read property 'test' of undefined". I'm running tests using testcafe runner and it was working fine before with 17.2, there is some related change which brought this regression or anything else? I tried to find all the changes but couldn't find exact pull request which might have regressed.

I have also tried by deleting entire node_modules, package-lock.json & pulling entire repo in another directory but still no luck!

### Command:
**node tests/testcafe/utils/testcafe-run.js -T Smoke**

```
jitesh.sojitra@pnq-jsojitra MINGW64 /c/project (develop)
$ node /c/project/tests/testcafe/utils/testcafe-run.js -T Smoke
Run command: --path C:\project\tests\testcafe\tests -T Smoke chrome -c 1 -r html-testrail -S -s C:\project\test-reports -p ${TEST} - ${QUARANTINE_ATTEMPT}.png

TypeError: Cannot read property 'test' of undefined
    at Object.keys.some.alias (C:\project\node_modules\testcafe-browser-tools\src\api\get-installations.js:61:24)
    at Array.some (<anonymous>)
    at addInstallation (C:\project\node_modules\testcafe-browser-tools\src\api\get-installations.js:58:30)
(node:16976) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

My Chrome is already latest Version 79.0.3945.130 (Official Build) (64-bit), i had updated it. I tried npm install again but still seeing same error.

![image](https://user-images.githubusercontent.com/21263826/72813583-9834ba80-3c89-11ea-87b7-81d8b7c86565.png)

Is this testcafe-browser-tools issue? It seems pulling 2.0.8-alpha.2 instead of 2.0.8 as i can see in node-modules and 2.0.8 is released from https://www.npmjs.com/package/testcafe-browser-tools an hour ago.

**Environment:**
Node: 11.15
TestCafe: 1.8.0
OS: Windows 10

**Comment by jiteshsojitra** [comment]
My Chrome is already latest Version 79.0.3945.130 (Official Build) (64-bit), i had updated it. I tried npm install again but still seeing same error.

![image](https://user-images.githubusercontent.com/21263826/72813583-9834ba80-3c89-11ea-87b7-81d8b7c86565.png)

Is this testcafe-browser-tools issue? It seems pulling 2.0.8-alpha.2 instead of 2.0.8 as i can see in node-modules and 2.0.8 is released from https://www.npmjs.com/package/testcafe-browser-tools an hour ago.

**Comment by bhj-patrik** [comment]
I have the same problem 

testcafe chrome test.js
Using locally installed version of TestCafe.
ERROR TypeError: Cannot read property 'test' of undefined
    at E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:61:24
    at Array.some (<anonymous>)
    at addInstallation (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:58:30)
    at async Promise.all (index 0)
    at searchInRegistry (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:97:5)
    at findWindowsBrowsers (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:103:39)
    at findBrowsers (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:170:16)
    at Object.default_1 [as default] (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-installations.js:223:30)
    at Object.default_1 [as getBrowserInfo] (E:\NodeProject\Node_exe\node_modules\testcafe-browser-tools\src\api\get-browser-info.js:24:27)
    at BrowserProviderPluginHost.isValidBrowserName (E:\NodeProject\Node_exe\node_modules\testcafe\src\browser\provider\built-in\dedicated\base.js:25:29)
    at BrowserProvider.isValidBrowserName (E:\NodeProject\Node_exe\node_modules\testcafe\src\browser\provider\index.ts:279:16)
    at Object.getBrowserInfo (E:\NodeProject\Node_exe\node_modules\testcafe\src\browser\provider\pool.js:108:14)
    at async Promise.all (index 0)
    at Bootstrapper._getBrowserInfo (E:\NodeProject\Node_exe\node_modules\testcafe\src\runner\bootstrapper.ts:206:29)
    at Bootstrapper.createRunnableConfiguration (E:\NodeProject\Node_exe\node_modules\testcafe\src\runner\bootstrapper.ts:421:29)

**Comment by AndreyBelym** [comment]
Thank you for your reports. Yes, we found a problem in the `testcafe-browser-tools` package and fixed it in `testcafe-browser-tools@2.0.8`. The fix is included with `testcafe@1.8.1-alpha.1`. You can install this prerelease version to check if the fix works in your environment. 

For housekeeping purposes, I mark this issue as a duplicate of #4661.
